### ISSUE TITLE
`items()` / `keys()` can return unboxed objects

### ISSUE THREAD

**Issue by mickael9** [cdgriffith/Box#40]
Test case:

```python
>>> import box
>>> box.__version__
'3.1.0'
>>> b = box.Box()
>>> b.foo = {}
>>> b.values()
dict_values([{}])
>>> b.foo # Force conversion by accessing the key
<Box: {}>
>>> b.values()
dict_values([<Box: {}>])

```

**Comment by cdgriffith** [comment]
Thanks for pointing this out! I will dig into this weekend if possible.

**Comment by cdgriffith** [comment]
Fixed in 3.1 https://github.com/cdgriffith/Box/commit/252e2f574213708cbbcd42bdec442434384127fe
